src/api/controllers/userController.js

```diff
@@ -0,0 +1,71 @@
+'use strict'
+const { generateAccessToken, generateRefreshToken, hashPassword } = require('/TokenGenerator')
+const userCollection = require('./errorControllers/index')
+
+const signin = async (req, res) => {
```

Comment:
the arguments are (req,res,next)
we must pass `next` to be able to use it in the catch block

same thing in the other handlers